### PR TITLE
feat: added resource section

### PR DESCRIPTION
**Pull request by Akshaybagai52**
**What kind of change does this PR introduce?**

**Issue Number:**

-  Related #282 
-  Related to #___ [](https://github.com/json-schema-org/website/pull/395)
-  Others? 

**Screenshots/videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

### CODE REVIEW

**Comment by benjagm** [comment]
What is the linked issue to this PR?

**Comment by Akshaybagai52** [comment]
> What is the linked issue to this PR?

https://github.com/json-schema-org/website/issues/282

**Comment by gregsdennis** [comment]
@Akshaybagai52 please update the opening comment. The issue links you have are formatted oddly.  Just use the PR number, e.g. `#282`

**Comment by benjagm** [comment]
There is still a lot of changes that are not required. Did you do a rebase?

**Review by benjagm — CHANGES_REQUESTED** [review]
This is looking really promising. 

You'll need to remove from this PR all the files unrelated to this feature. Have you rebased?

In addition: 

- I'll suggest reduce the space between every item of the list.
- Change the icon for the sidebar section of resources. Maybe something like this: https://icons.getbootstrap.com/icons/bookshelf/
- There is a strange effect when navigating that will require some attention. I'll record a video. This is the [video](https://github.com/json-schema-org/website/pull/509#issuecomment-1989253444).

**Comment by benjagm** [comment]
The promised video with the strange behaviour when navigating between different resources and different sections of the sidebar.


https://github.com/json-schema-org/website/assets/40007659/ed337080-c16b-4630-b581-9300705aa093

File: pages/resources/[slug].page.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import React, { useEffect, useState } from 'react';
```
[the remark this answers is not in the document]
Comment (Akshaybagai52) [reply pages/resources/[slug].page.tsx:1]: actually this is dynamic route so all the things depend on the value of slug which I'm getting from the route is it good to rename it to index file?

**Comment by Akshaybagai52** [comment]
> There is still a lot of changes that are not required. Did you do a rebase?

These changes are automatically coming whenever i'm changing base repo

**Comment by DhairyaMajmudar** [comment]
I think its because you haven't synced the branch with the current branch.

**Comment by Akshaybagai52** [comment]
> I think its because you haven't synced the branch with the current branch.

Yeah I think I need to rebase the branch

File: pages/resources/[slug].page.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (benjagm) [reply pages/resources/[slug].page.tsx:1]: > actually this is dynamic route so all the things depend on the value of slug which I'm getting from the route is it good to rename it to index file?

You are right. Let's leave this way thanks!

**Comment by benjagm** [comment]
Thanks @Akshaybagai52 Please let me know when you need me to check this again.

**Comment by Akshaybagai52** [comment]
> Thanks @Akshaybagai52 Please let me know when you need me to check this again.

It is completed you can review now

**Comment by Akshaybagai52** [comment]
Hello @benjagm did you review this PR ?

**Review by benjagm — APPROVED** [review]
Amazing job here!! I pushed some changes to make sure it merges smoothly into the dev branch! Excellent Job!!

**Comment by Akshaybagai52** [comment]
> Amazing job here!! I pushed some changes to make sure it merges smoothly into the dev branch! Excellent Job!!

Thankyou ✨

**Pull request merged**